src/app/modules/shared/components/header/header.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+    selector: 'mto-header',
+    templateUrl: './header.component.html',
+    styleUrls: ['./header.component.scss']
+  })
+  export class HeaderComponent {
+    @Input() isLoggedIn = false;
+    @Input() title = 'Example Title';
+    @Input() userPicture = 'assets/img/profile_picture.jpg';
+  }
+  
```

Comment:
Whitespace!

Reply:
It's tslint rule
https://palantir.github.io/tslint/rules/whitespace/

Here why:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline